include/readdy/api/SimulationLoop.h

```diff
@@ -200,9 +202,25 @@ class SimulationLoop {
         configGroup = std::make_unique<h5rd::Group>(file.createGroup("readdy/config"));
     }
 
-    scalar &skinSize() { return _skinSize; }
+    scalar &neighborListDistance() {
```

Comment:
I'd rather call this `neighborListCutoff` or something similar.. not sure what a neighbor list distance is

Reply:
agreed